servers.json

```diff
@@ -6,6 +6,13 @@
             "edenmine.eu"
         ]
     },
+    {
+        "id": "Skirmish"
```

Comment:
You must add a `,` after the id field,

you should also make the id lowercase like all other entries